__tests__/news/news.query.test.js

```diff
@@ -1,126 +1,274 @@
 /* eslint-disable no-undef */
 const { gql } = require('apollo-boost');
-const client = require('../../utils/apollo-client');
-const { newsQuery } = require('../../modules/news/news.resolver');
+const client = require('../../utils/apollo-test-client');
 require('dotenv').config();
-const newsService = require('../../modules/news/news.service');
 
-const NEWS_ID = '5f25bf7a6f5b2c3a189eaf2a';
+let NEWS_ID = '';
```

Comment:
There is no need to uppercase this variable, it is not a constant, it is going to be changed anyway and it exists only for the purpose of the test, so no need to uppercase it. The same thing for the next one

Reply:
fixed